docs/getting_started.md

```diff
@@ -0,0 +1,22 @@
+# Getting Started
+
+Follow the steps below to get started:
+
+1. Visit the [RapidRails Github](https://github.com/danielpaul/RapidRails).
+
+2. Click the **Use this template** button and select **Create a new repository**.
+
+   ![Use template](use_template.png)
+
+3. Fill out the fields and click the **Create repository from template** button.
+
+   ![Use template](create_repo.png)
+
+4. Clone the repository to your local machine.
+
+5. Navigate to the repository on your terminal and run `bin/setup`.
+
+6. Follow the prompts from the setup script to configure your application.
```

Comment:
add that they can read more about configuration in the next section where we go in depth about the setup and things that they need to configure.